website/docs/guides/explorer-qs.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Quickstart for the dbt Explorer workshop"
+description: "Use this guide to build and define metrics, set up the dbt Cloud Semantic Layer, and query them using Google Sheets."
+sidebar_label: "Quickstart dbt Explorer"
+icon: 'guides'
+hide_table_of_contents: true
+tags: ['Explorer', 'Snowflake', 'dbt Cloud','Quickstart']
+keywords: ['dbt Explorer','Mesh','dbt Cloud', 'Snowflake', 'Multi-Project']
+level: 'Beginner'
+recently_updated: true
+---
+
+
+## Introduction
+
+Unlock the power of [dbt Explorer](/docs/collaborate/explore-projects) in this hands-on workshop designed for analytics engineers, data analysts, stakeholders, and data leaders.
+
+This quickstart guide accompanies the Explorer hands-on workshop and help you dive into a production-level dbt Mesh implementation and discover how to explore your data workflows.⁠ Whether you're looking to streamline your data operations, improve data quality, or self-serve information about your data platform, this workshop will equip you with the tools and knowledge to take your dbt projects to the next level.
+
+By the end of the guide and workshop, you'll understand how to leverage dbt Explorer and have the confidence to navigate multiple dbt projects, trace dependencies, and identify opportunities to improve performance and data quality.
+
+### What you'll learn
+In this guide, you will learn how to:
+- Navigate multiple dbt projects using dbt Explorer
+- Self-serve on data documentation
+- Trace dependencies at the model and column level
+- Identify opportunities to improve performance and data quality
+
+### Prerequisites
+- Familiarity with data platforms
+
+## Setup
+Now we’ll be creating your dbt Cloud account and connecting it to a data warehouse. 
+- Go to this URL:  https://cloud.getdbt.com/coalesce-workshop-signup
+  - If you are already signed into dbt Cloud, sign out first!
+  - Enter your first name and last name
+  - Choose the **Exploring a dbt Mesh implementation with dbt Explorer** option.
+  - Enter the password provided by the workshop facilitator
+- Agree to the terms of service and click the **Complete Registration** button.
+- Wait about 30 seconds, and you’ll be in the dbt Cloud project for this course, already connected to a data warehouse!
+
+## Get and overview of your project and view the status of your latest runs
+
+   <Lightbox src="/img/quickstarts/dbt-cloud/explorer_performance_tab.png" title="dbt Explorer performance tab" />
+
+With dbt Explorer, you can view your project's resources (such as models, tests, and metrics), their lineage, and model consumption to gain a better understanding of its latest production state. Navigate and manage your projects within dbt Cloud to help you and other data developers, analysts, and consumers discover and leverage your dbt resources.
+
+#### Hands-On
+- Explore the Model tab 
+  - Which job took the Longest? 
```

Comment:
how do you feel about 'Try it out' so it's more clear that this is an interactive part?